### ISSUE TITLE
Source or automate build of UAA war

### ISSUE THREAD

**Issue by svenaas** [18F/federalist#3749]
This repo no longer includes `/uaa/cloudfoundry-identity-uaa-4.19.0.war` due to an included vulnerability that we don't want running on the staging environment (the UAA container only needs to be run in local development).

In #3744, from which this issue is extracted, we removed the war file from the repo and established a way for developers to use their own local copy. What we do not have is place for developers to _get_ that local copy. 

If we can find a reliable, canonical (or at least reputable) source online, whether of `cloudfoundry-identity-uaa-4.19.0.war` specifically or of a version sufficiently compatible with our codebase, we can simply reference that in the README. 

Failing that, we can try building the war locally ([doing this is covered in the Cloud Foundry documentation](https://docs.cloudfoundry.org/concepts/architecture/uaa.html)). If that works it may be practical to automate such a build when Docker prepares the UAA container.

**Comment by svenaas** [comment]
Mind you, the Cloud Foundry docs tell you to 
```
git clone git://github.com/cloudfoundry/uaa.git
```
instead of the current and correct
```
git clone git@github.com:cloudfoundry/uaa.git
```
so they _may_ not be reliable for our purposes...

**Comment by svenaas** [comment]
After installing Java:
```
brew tap homebrew/cask-versions
brew update
brew tap homebrew/cask
brew install java
sudo ln -sfn /usr/local/opt/openjdk/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk.jdk
```
I tried Cloud Foundry's
```
./gradlew :cloudfoundry-identity-uaa:war
```
and got errors that may be attributable to Java version issues — the steps above installed Java 17.0.1. Trying again with Java8 specifically: 
```
brew install adoptopenjdk8
export JAVA_HOME=$(/usr/libexec/java_home -v1.8)
./gradlew :cloudfoundry-identity-uaa:war
```
This time Gradle told me it requires Java11 at least. _Thanks_, Gradle. 

```
brew install openjdk@11 
brew uninstall openjdk@17
sudo ln -sfn /usr/local/opt/openjdk@11/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk-11.jdk
export JAVA_HOME=$(/usr/libexec/java_home -v11)
./gradlew :cloudfoundry-identity-uaa:war
```
and I got
```
BUILD SUCCESSFUL in 1m 2s
```
but so far I haven't figured out where it _put_ the war file.

**Comment by svenaas** [comment]
Ah. It generated
```
/uaa/uaa/build/libs/cloudfoundry-identity-uaa-0.0.0.war
```
Not sure about that `0.0.0`. It's a 63M file, as compared with the 55M `cloudfoundry-identity-uaa-4.19.0.war` we used to have here.

**Comment by svenaas** [comment]
This works to produce a 75.13.0 war (thanks, @pburkholder!):
```
wget https://github.com/cloudfoundry/uaa/archive/refs/tags/v75.13.0.tar.gz
tar -zxvf v75.13.0.tar.gz
cd uaa-75.13.0/
VERSION=75.13.0
./gradlew :clean :assemble -Pversion=${VERSION}
```

Given changes in `Dockerfile-uaa` (to require `openjdk:11.0.13-jre` and Tomcat 9.0.45 and to copy in the different war) the resulting war file does suffice to log in to Federalist on my local workstations. 

My next step was going to be seeing whether I could automate the above build in the Dockerfile. I got as far as 
```
RUN wget https://github.com/cloudfoundry/uaa/archive/refs/tags/v75.13.0.tar.gz
RUN tar -zxvf v75.13.0.tar.gz
RUN cd uaa-75.13.0/
RUN ./gradlew :clean :assemble -Pversion=75.13.0
```
before I realized that that last step was going to be a pain: it requires Java 11 and Gradle either on the development workstation or leveraged one way or another using Docker (with the openjdk container? with another container just for the build?).

So now I'm planning to grab the Stark & Wayne's pre-built war, hosted on GitHub at https://github.com/starkandwayne/uaa-war-releases/releases.
